### PR TITLE
Passed keyProp to Link and Node righClick handler

### PR DESCRIPTION
**Pull request by Linton-Samuel-Dawson**
passed keyProp to onContextMenu

### CODE REVIEW

**Comment by jpb12** [comment]
Can you fix the `[build errors](https://travis-ci.org/jpb12/react-tree-graph)`?

**Comment by Linton-Samuel-Dawson** [comment]
@jpb12 done

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 language: node_js
 node_js:
-  - "8"
-  - "10"
+  - "stable"
```
[the remark this answers is not in the document]
Comment (Linton-Samuel-Dawson) [reply .travis.yml]: I hadn't used Travis before, so I copy paste from tutorials

[the remark this answers is not in the document]
Comment (jpb12) [reply .travis.yml]: Can you revert them then? I don't think there was anything wrong with the existing config.

**Pull request merged**